scanner/src/lib/dataproducts/Product_FIBC_ProductDataSheet_v0.1.svelte

```diff
@@ -0,0 +1,176 @@
+  import Divider from "$components/Divider/index.svelte"
+
+  enum FIBCType {
+    A = "A",
+    B = "B",
+    C = "C",
+    D = "D",
+  }
+
+  enum FillingMethod {
+    TOP = "Top",
+    BOTTOM = "Bottom",
+  }
+
+  export let data: {
+    model: string
+    type: FIBCType
+    safeWorkingLoad: number
+    fillingMethod: FillingMethod
+    uvProtected: boolean
+    uvGuaranteeYears: number
+    materialComposition: string
+    manufacturingInformation: {
+      manufacturer: string
+      batchNumber: string
+      manufacturingDate: string
+      expirationDate: string
+      productionCity: string
+      productionCountry: string
+      safetyFactor: string
+      complianceCertifications: string[]
+    }
+    dimensions: {
+      externalWidth: number
+      externalLength: number
+      externalHeight: number
+      internalWidth: number
+      internalLength: number
+      internalHeight: number
+      volume: number
+    }
+    loops: {
+      type: string
+      height: number
+      color: string
+    }
+    body: {
+      stitching: string
+      stitchingColor: string
+      coatingApplied: boolean
+      color: string
+    }
+    topSpout: {
+      diameter: number
+      length: number
+      coatingApplied: boolean
+      color: string
+    }
+    bottom: {
+      diameter: number
+      length: number
+      coatingApplied: boolean
+      color: string
+    }
+    liner: {
+      type: string
+      thickness: number
+      color: string
+    }
+    documentPocket: string
+    handlingInstructions: string
+    inspectionDate: string
+    typeMarkings: string[]
+    foodSafe: boolean
+  }
+
+  const { manufacturingInformation, dimensions, loops, body, topSpout, bottom, liner } = data
+</script>
+
+<Article>
+  <DataRow label="Model" value={data.model} />
+  <DataRow label="Type" value={data.type} />
+  <DataRow label="Safe working load" value={formatNumber(data.safeWorkingLoad, "kg")} />
+  <DataRow label="Filling method" value={data.fillingMethod} />
+  <DataRow label="UV protected" value={data.uvProtected} />
+  <DataRow
+    label="UV guarantee"
+    value={formatNumber(data.uvGuaranteeYears, getYearLabel(data.uvGuaranteeYears))}
+  />
+  <DataRow label="Material composition" value={data.materialComposition} />
+  <Divider />
+  <SectionHeader title="Manufacturing information">
+    The manufacturing details of the bag.
+  </SectionHeader>
+  <DataRow label="Manufacturer" value={manufacturingInformation.manufacturer} />
+  <DataRow label="Batch number" value={manufacturingInformation.batchNumber} />
+  <DataRow
+    label="Manufacturing date"
+    value={localizeDate(manufacturingInformation.manufacturingDate)}
+  />
+  <DataRow label="Expiration date" value={localizeDate(manufacturingInformation.expirationDate)} />
+  <DataRow label="Production city" value={manufacturingInformation.productionCity} />
+  <DataRow
+    label="Production country"
+    value={countryListAlpha3[manufacturingInformation.productionCountry]}
+  />
```

Comment:
This kind of stuff would be good to summarize as "Produced in: {city}, {country}"